### ISSUE TITLE
Simple-Ontology

### ISSUE THREAD

**Issue by alFrie** [BAMresearch/LebeDigital#107]
In order to be able to write a working mapping script, Mattheo provided us with his personal minimum mapping example. It can be found in lebedigital/SimpleOntology on the Simple-Ontology branch.

The tasks are now:

- [x] converting the synthetic data in Data.xlsx to yaml files, one for each specimen.
- [x] accessing the content of the .ttl file with Python
- [x] writing a script that replaces the placeholders within the SimpleOntology.ttl with the metadata
- [x] comparing the result to Mattheos result (SimpleOntologyMapped.ttl) → for this we have the problem that the word "length" and "diameter" are misspelled, so keep this in mind when comparing the two files or just keep it spelled like this.

**Comment by joergfunger** [comment]
I looked into the example that is given in the SimpleOntology.ttl. Some remarks
- please rename the branch to match the issue as we have done with all other branches. This way, it is easier to find that branch and relate it to the work that is done
- why are the used namespaces ns1 and ns2 different, that is weird. What is that link (http://w3id.com/concrete or http://w3id.org/concrete) referencing to? ns1 seems not to be used apart from the ontology definition 
- how do we distinguish placeholders from comments? I would suggest replacing "Diametre_Value" by "$$Diameter_Value$$ (correct spelling), thus "everything that is within "$$somePlaceholder$$" is to be replaced

**Comment by joergfunger** [comment]
As for the synthetic data, directly start with the yaml files, there is no need to convert xlxs files first for testing the mapping.

**Comment by eriktamsen** [comment]
To answer your question @alFrie: I have not worked with ttl files. As far as I know, the rdflib package should be able to do that.

**Comment by mattheokru** [comment]
_why are the used namespaces ns1 and ns2 different, that is weird. What is that link (http://w3id.com/concrete or http://w3id.org/concrete) referencing to? ns1 seems not to be used apart from the ontology definition_

Like @alFrie said it's just a simple example and the link to http://w3id.com/concrete was something Juliette and me tried to do when creating taxonomie ontology. However the Link doesn't work and was only used because it was still set as a standard in Protege. I can change it to something like http://www.semanticweb.org/mkrueger/ontologies/2023/5/SimpleOntology which is the standard way in protege however I don't think it makes a difference in this simple example

**Comment by eriktamsen** [comment]
Well, as you @alFrie/@mattheokru are looking for an answer asap, I will try to answer with my opinion, as Jörg is on vacation this week and will probably not answer before Monday. 
From my understanding, I would say it does not matter as long as we can access the data. I thought it should not make a difference, as long as it is possible to link the data, so that we can querry e.g. hydration data for a specific mix for which we also have youngs modulus data.
In conclusion: I would say, you can decide which ever seems to be less work in the long run, as long as the functionality is the the same.

**Comment by joergfunger** [comment]
Please talk to Stephan or Thilo about that. In my opinion, we would like to map each step individually. If you only have E-modulus measured, there is no need to map the heat of hydration, and similarly, if you have a heat of hydration test, there is not necessarily a Emodulus test. In addition, those might be separated in time (e.g. first you map the mix composition) and then 28 days later (when the tests are performed) the compression/Youngs modulus data.
